lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Player::Human < Player
+
+  def move(board)
+    puts "Enter a position 1-9: "
+    pos = gets.chomp
+    return pos
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Enter a position 1-9: "
  gets.chomp
end
```